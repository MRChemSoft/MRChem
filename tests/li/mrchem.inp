# vim:syntax=sh:

abs_prec = 1.0e-3                 # Final precision in energy

MRA {
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
multiplicity = 2
$coords
Li   0.0     0.0      0.0
$end
}

WaveFunction {
  method = HF                     # Wave function method (HF or DFT)
  restricted = false
}

Properties {
  scf_energy = true               # Compute ground state energy
}

SCF {
  kain = 3                        # Length of KAIN iterative history
  rotation = 5                    # Number of iterations between diag/loc
  initial_guess = GTO             # Type of initial guess (none, gto, mw)
}
