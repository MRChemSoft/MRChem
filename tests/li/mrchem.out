


************************************************************
***                                                      ***
***    MRChem 1.0.0 (rev. e6faeac)                       ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Jonas Juselius   <jonas.juselius@uit.no>          ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***                                                      ***
************************************************************


Print level  : 0

BLAS was NOT found, Eigen will be used instead!

+++ Parallel execution: 
  MPI hosts available     : 1
  Threads/host            : 16
  Total used CPUs         : 16



============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.00000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [   -32.00000   -32.00000   -32.00000 ]
 upper bounds     = [    32.00000    32.00000    32.00000 ]
 total length     = [    64.00000    64.00000    64.00000 ]
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   6.41500e-04   2.25611e-03
============================================================


============================================================
         Setting up occupied orbitals (open-shell)
------------------------------------------------------------
    0    a    1                       9.999999977511659e-01
    1    a    1                       9.999999999038982e-01
    2    b    1                       9.999999977532259e-01
------------------------------------------------------------
                 Wall time: 3.66540e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     904      1.94530e-02
 Coulomb density                       456      4.62420e-02
 Coulomb potential                     328      3.28233e-01
 Hartree-Fock exchange                 392      1.41939e+00
------------------------------------------------------------
                 Wall time: 1.81347e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              6.41500e-04
------------------------------------------------------------
 Orbital threshold                              8.00937e-03
 Property threshold                             6.41500e-05
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.63300e-03
 Diagonalizing matrix                           7.00000e-06
 Rotating orbitals                              2.18510e-02
------------------------------------------------------------
                 Wall time: 2.45400e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.378892605226736e+00
------------------------------------------------------------
 Total energy                        -7.378892605226736e+00
------------------------------------------------------------
                 Wall time: 7.62380e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -2.45924e+00
 Creating operator with lambda                 -1.94378e-01
 Creating operator with lambda                 -2.44260e+00
------------------------------------------------------------
                 Wall time: 3.35679e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  1.00204e+00  0.00000e+00   392     0  2.0e-03  3.8e-01
  1  1.00182e+00  0.00000e+00   328     0  1.8e-03  4.0e-01
  2  1.00331e+00  0.00000e+00   392     0  3.3e-03  3.7e-01
------------------------------------------------------------
                 Wall time: 1.14250e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.10000e-05
 Copy orbitals                                  8.42300e-03
 Copy orbital updates                           8.98000e-03
------------------------------------------------------------
                 Wall time: 1.74990e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      6.41500e-04
------------------------------------------------------------
 Nuclear potential                     840      1.75330e-02
 Coulomb density                       456      3.99020e-02
 Coulomb potential                     264      2.38956e-01
 Hartree-Fock exchange                 456      1.01140e+00
------------------------------------------------------------
                 Wall time: 1.30797e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.48987e+00   3.26090e-02        456    a    1    0
  1  -1.96281e-01   3.45322e-02        392    a    1    0
  2  -2.47301e+00   3.31554e-02        456    b    1    0
------------------------------------------------------------
 Total error:                            5.79232e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.373454007343672e+00     7.37345e+00    0
 N-E          -1.703502630763298e+01    -1.70350e+01    0
 Coulomb       4.061963811582776e+00     4.06196e+00    0
 Exchange     -1.779284116520204e+00    -1.77928e+00    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.378892605226736e+00    -7.37889e+00    0
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.378892605226736e+00    -7.37889e+00    0
============================================================


################ Wall time: 3.23978e+00 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              6.41500e-04
------------------------------------------------------------
 Orbital threshold                              8.00937e-03
 Property threshold                             6.41500e-05
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.95000e-03
 Diagonalizing matrix                           7.00000e-06
 Rotating orbitals                              1.90680e-02
------------------------------------------------------------
                 Wall time: 2.20800e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.431035589541128e+00
------------------------------------------------------------
 Total energy                        -7.431035589541128e+00
------------------------------------------------------------
                 Wall time: 7.30240e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -2.49000e+00
 Creating operator with lambda                 -1.96154e-01
 Creating operator with lambda                 -2.47301e+00
------------------------------------------------------------
                 Wall time: 4.20353e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.97547e-01  0.00000e+00   456     0  2.5e-03  4.3e-01
  1  9.99022e-01  0.00000e+00   328     0  9.8e-04  3.6e-01
  2  9.97924e-01  0.00000e+00   456     0  2.1e-03  4.1e-01
------------------------------------------------------------
                 Wall time: 1.19408e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             7.00000e-06
 Copy orbitals                                  7.54600e-03
 Copy orbital updates                           1.02460e-02
------------------------------------------------------------
                 Wall time: 1.79140e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      6.41500e-04
------------------------------------------------------------
 Nuclear potential                     840      1.72440e-02
 Coulomb density                       520      4.41520e-02
 Coulomb potential                     264      2.44193e-01
 Hartree-Fock exchange                 520      9.61752e-01
------------------------------------------------------------
                 Wall time: 1.26758e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.48883e+00   4.12555e-03        520    a    1    1
  1  -1.96228e-01   1.22193e-02        392    a    1    0
  2  -2.47141e+00   3.76883e-03        520    b    1    1
------------------------------------------------------------
 Total error:                            1.34364e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.337855047619378e+00    -3.55990e-02    0
 N-E          -1.704076670083043e+01    -5.74039e-03    0
 Coulomb       4.045069621091996e+00    -1.68942e-02    0
 Exchange     -1.773193557422071e+00     6.09056e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.431035589541128e+00    -5.21430e-02    0
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.431035589541128e+00    -5.21430e-02    0
============================================================


################ Wall time: 3.36159e+00 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              6.41500e-04
------------------------------------------------------------
 Orbital threshold                              8.00937e-03
 Property threshold                             6.41500e-05
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.432514146738305e+00
------------------------------------------------------------
 Total energy                        -7.432514146738305e+00
------------------------------------------------------------
                 Wall time: 8.04880e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -2.48883e+00
 Creating operator with lambda                 -1.96228e-01
 Creating operator with lambda                 -2.47141e+00
------------------------------------------------------------
                 Wall time: 3.39953e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.98900e-01  0.00000e+00   456     0  1.1e-03  4.3e-01
  1  9.99432e-01  0.00000e+00   328     0  5.7e-04  4.0e-01
  2  9.99071e-01  0.00000e+00   456     0  9.3e-04  4.3e-01
------------------------------------------------------------
                 Wall time: 1.25063e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             8.00000e-06
 Setup linear system                            2.69850e-02
 Solve linear system                            1.50000e-05
 Expand solution                                3.42170e-02
------------------------------------------------------------
                 Wall time: 6.13600e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      6.41500e-04
------------------------------------------------------------
 Nuclear potential                     840      1.73810e-02
 Coulomb density                       520      4.39080e-02
 Coulomb potential                     264      2.43471e-01
 Hartree-Fock exchange                 520      9.64196e-01
------------------------------------------------------------
                 Wall time: 1.26920e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.48713e+00   1.96169e-03        520    a    1    1
  1  -1.96214e-01   7.86707e-03        392    a    1    1
  2  -2.46935e+00   1.53032e-03        520    b    1    1
------------------------------------------------------------
 Total error:                            8.25111e-03    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.392529590931247e+00     5.46745e-02    0
 N-E          -1.710108900942979e+01    -6.03223e-02    0
 Coulomb       4.053408299011076e+00     8.33868e-03    0
 Exchange     -1.777363027250836e+00    -4.16947e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.432514146738305e+00    -1.47856e-03    0
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.432514146738305e+00    -1.47856e-03    0
============================================================


################ Wall time: 3.36605e+00 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              6.41500e-04
------------------------------------------------------------
 Orbital threshold                              8.00937e-03
 Property threshold                             6.41500e-05
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.432593822737678e+00
------------------------------------------------------------
 Total energy                        -7.432593822737678e+00
------------------------------------------------------------
                 Wall time: 7.98940e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -2.48713e+00
 Re-using operator with lambda                 -1.96228e-01
 Creating operator with lambda                 -2.46935e+00
------------------------------------------------------------
                 Wall time: 2.23921e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99932e-01  0.00000e+00   456     0  6.8e-05  4.2e-01
  1  9.99949e-01  0.00000e+00   328     0  5.1e-05  3.7e-01
  2  9.99960e-01  0.00000e+00   456     0  4.0e-05  4.1e-01
------------------------------------------------------------
                 Wall time: 1.20719e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             3.38900e-03
 Setup linear system                            8.04190e-02
 Solve linear system                            1.40000e-05
 Expand solution                                5.11370e-02
------------------------------------------------------------
                 Wall time: 1.35109e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      6.41500e-04
------------------------------------------------------------
 Nuclear potential                     840      1.73270e-02
 Coulomb density                       520      4.40240e-02
 Coulomb potential                     264      2.38430e-01
 Hartree-Fock exchange                 520      9.39790e-01
------------------------------------------------------------
                 Wall time: 1.23982e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.48682e+00   2.64206e-04        520    a    1    1
  1  -1.96230e-01   1.15746e-03        392    a    1    1
  2  -2.46895e+00   2.02139e-04        520    b    1    1
------------------------------------------------------------
 Total error:                            1.20431e-03    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.431327009131607e+00     3.87974e-02    0
 N-E          -1.714381751548984e+01    -4.27285e-02    0
 Coulomb       4.060666941172561e+00     7.25864e-03    0
 Exchange     -1.780770257552008e+00    -3.40723e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.432593822737678e+00    -7.96760e-05    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.432593822737678e+00    -7.96760e-05    1
============================================================


################ Wall time: 3.24712e+00 sec ################


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        3.37900e-03
 Diagonalizing matrix                           9.00000e-06
 Rotating orbitals                              1.04740e-02
------------------------------------------------------------
                 Wall time: 1.39520e-02 sec 
============================================================


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    5.79232e-02    -7.378892605226736e+00   -7.37889e+00
  2    1.34364e-02    -7.431035589541128e+00   -5.21430e-02
  3    8.25111e-03    -7.432514146738305e+00   -1.47856e-03
  4    1.20431e-03    -7.432593822737678e+00   -7.96760e-05
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      6.41500e-04
------------------------------------------------------------
 Nuclear potential                     840      1.79710e-02
 Coulomb density                       520      4.42850e-02
 Coulomb potential                     264      2.41212e-01
 Hartree-Fock exchange                 520      9.37878e-01
------------------------------------------------------------
                 Wall time: 1.24156e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 7.95040e-02 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-2.48683e+00 0.00000e+00 0.00000e+00
0.00000e+00 -1.96225e-01 0.00000e+00
0.00000e+00 0.00000e+00 -2.46895e+00
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -2.486829288651525e+00
    1    a    1                      -2.468948358816514e+00
    2    b    1                      -1.962252628973103e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -5.152002910365349e+00
 Kinetic energy:                      7.434089642314047e+00
 E-N energy:                         -1.714730810760367e+01
 Coulomb energy:                      4.061793417457687e+00
 Exchange energy:                    -1.781185639995551e+00
 X-C energy:                          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy                   -7.432610687827484e+00
 Nuclear energy                       0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.432610687827484e+00
                    (kJ/mol)         -1.951431653649900e+04
                    (kcal/mol)       -4.664033586002349e+03
                    (eV)             -2.022516313532190e+02
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 1.688406e+01                ***
***                                                      ***
************************************************************


